<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskHoleParameters</class>
 <widget class="QWidget" name="TaskHoleParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Hole Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="7">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>&lt;b&gt;Threading and size&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="4">
    <widget class="Gui::PrefQuantitySpinBox" name="ThreadAngle" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Countersink angle</string>
     </property>
    </widget>
   </item>
   <item row="8" column="5" colspan="2">
    <widget class="Gui::PrefQuantitySpinBox" name="Diameter" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="4" colspan="3">
    <widget class="Gui::PrefQuantitySpinBox" name="HoleCutDepth" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QComboBox" name="DepthType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Dimension</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Through all</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="16" column="2" colspan="2">
    <widget class="Gui::PrefQuantitySpinBox" name="TaperedAngle" native="true"/>
   </item>
   <item row="1" column="3" colspan="4">
    <widget class="Gui::PrefQuantitySpinBox" name="ThreadPitch" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="5">
    <widget class="QComboBox" name="ThreadSize">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="5">
    <widget class="QComboBox" name="HoleCutType">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Diameter</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="5">
    <widget class="QComboBox" name="ThreadFit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Standard fit</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Close fit</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QComboBox" name="ThreadClass">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="7">
    <widget class="QLabel" name="label_14">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&lt;b&gt;Hole cut&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Fit</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Depth</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="5">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="9" column="5" colspan="2">
    <widget class="Gui::PrefQuantitySpinBox" name="Depth" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="4">
    <widget class="Gui::PrefQuantitySpinBox" name="ThreadCutOffOuter" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="QLabel" name="label_11">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Diameter</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="4">
    <widget class="Gui::PrefQuantitySpinBox" name="ThreadCutOffInner" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Depth</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="12" column="4" colspan="3">
    <widget class="Gui::PrefQuantitySpinBox" name="HoleCutDiameter" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="2" colspan="5">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="Gui::PrefQuantitySpinBox" name="DrillPointAngle" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="14" column="4" colspan="3">
    <widget class="Gui::PrefQuantitySpinBox" name="HoleCutCountersinkAngle" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::PrefQuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
